packages/firebase_admob/ios/Classes/FirebaseAdMobPlugin.m

```diff
@@ -185,7 +195,7 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
                 call:call
               result:result];
   } else if ([call.method isEqualToString:@"showAd"]) {
-    [self callShowAd:mobileAdId call:call result:result];
+    [self callShowAd:mobileAdId call:call params:call.arguments result:result];
```

Comment:
The `call` parameter already carries the arguments. No need to pass that as a separate `params`.